### PR TITLE
minor fix about visibility in reference

### PR DESCRIPTION
**Pull request by liigo**

### CODE REVIEW

**Comment by KalitaAlexey** [comment]
You mentioned that trait items are public by default. Can I make a trait item private?

**Comment by liigo** [comment]
you can't, at least currently, we don't use `priv` keyword any more.

File: src/doc/reference.md [hunk 1]
```diff
@@ -1730,7 +1730,8 @@ of an item to see whether it should be allowed or not. This is where privacy
 warnings are generated, or otherwise "you used a private item of another module
 and weren't allowed to."
 
-By default, everything in Rust is *private*, with one exception. Enum variants
+By default, everything in Rust is *private*, with two exceptions: Associated
+items of a Trait are public by default; Enum variants
```
Comment (petrochenkov) [inline src/doc/reference.md]: Associated items and variants are not public by default, they inherit visibility from their traits and enums respectively. E.g.
```
pub(vis) trait Tr {
    fn f();
}
```
is "desugared" into
```
pub(vis) trait Tr {
    pub(vis) fn f(); // (Not syntactically legal)
}
```

**Comment by steveklabnik** [comment]
Looks like @petrochenkov 's feedback needs to be addressed.

**Comment by liigo** [comment]
done.

**Pull request merged**